src/events.rs

```diff
@@ -101,6 +101,10 @@ pub enum WindowEvent {
 pub enum DeviceEvent {
     Added,
     Removed,
+
+    /// In the case of mouse motion, the axis is 0 (in x) or 1 (in y), and the value is the distance delta.
+    /// For example, repeatedly moving the mouse 5 pixels to the left will generate a sequence of
```

Comment:
There's actually no correlation between the 'motion' events and how much the mouse actually moves on screen. Mouse movement on the screen is different than the actual movement of the mouse!